odoo_module_migrate/migration_scripts/migrate_120_130.py

```diff
@@ -28,5 +44,6 @@
     },
     ".xml": {
         r"( |\t)*<field name=('|\")view_type('|\")>.*</field>\n": "",
+        r"src_model": "binding_model",
```

Comment:
It's already part of master branch. Please remove from commit diff (via ``git rebase -i HEAD~2``)